mlcp/src/main/java/com/marklogic/contentpump/ConfigConstants.java

```diff
@@ -134,6 +134,18 @@
     static final String SSL_PROTOCOL = "ssl_protocol";
     static final String INPUT_SSL_PROTOCOL = "input_ssl_protocol";
     static final String OUTPUT_SSL_PROTOCOL = "output_ssl_protocol";
+    static final String KEYSTORE_PATH ="keystore_path";
+    static final String KEYSTORE_PASSWD ="keystore_passwd";
```

Comment:
Zhichun,

Could you please change all "passwd" to "password"? Otherwise the password masking wouldn't work. (See https://bugtrack.marklogic.com/53601)

Reply:
sure